packages/app-project/pages/projects/[owner]/[project]/logger.js

```diff
@@ -38,7 +38,7 @@ class Logger extends Component {
           </Heading>
           <div>
             <p>
-              You're in development mode (<code>NODE_ENV !== 'production'</code>), so you're seeing this message. In production, this route will 404, so make sure you check the CloudFront config so it's picked up by PFE.
+              You are in development mode (<code>NODE_ENV !== &#39;production&#39;</code>), so you are seeing this message. In production, this route will 404, so make sure you check the CloudFront config so it is picked up by PFE.
```

Comment:
Does the linter not like contractions? 😆

Reply:
I thought about encoding the apostrophe but I have come across volunteers, who don’t speak English, who find contractions hard to understand.

Reply:
Also not sure why unencoded single quotes break the build in Next 11. That seemed a bit severe.